concourse/tasks/remove-healthcheck-db.yml

```
@@ -13,11 +13,17 @@ run:
     - |
       ./paas-cf/concourse/scripts/import_bosh_ca.sh
       . ./config/config.sh
-      if curl -I -f $API_ENDPOINT/info; then
-        echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
-        cf target -o testers -s healthcheck
-        cf delete healthcheck -f -r
-        if cf services | grep -q healthcheck-db; then
-          cf delete-service healthcheck-db -f
-        fi
+      if ! curl -I -f $API_ENDPOINT/info; then
+        echo "CF API unavailable. Skipping..."
+        exit 0
+      fi
+      echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+      if ! cf org testers > /dev/null; then
```

Well, both. I was getting a bit confused.

@combor's point about silent failure if we change the org name is correct. I think it's probably OK to make this change for the moment, but raises the importance of https://www.pivotaltracker.com/story/show/120002281